packages/x-date-pickers/src/internals/hooks/useField/useFieldState.ts

```diff
@@ -254,20 +254,16 @@ export const useFieldState = <
     }
 
     const activeSection = state.sections[selectedSectionIndexes.startIndex];
-
-    if (activeSection.value === '') {
-      return;
-    }
-
     const activeDateManager = fieldValueManager.getActiveDateManager(utils, state, activeSection);
 
     const nonEmptySectionCountBefore = activeDateManager
       .getSections(state.sections)
       .filter((section) => section.value !== '').length;
-    const isTheOnlyNonEmptySection = nonEmptySectionCountBefore === 1;
+    const hasNoOtherNonEmptySections =
```

Comment:
Small fix when the active section is already empty
If could never happen before